src/components/hand-poses.js

```diff
@@ -36,7 +36,13 @@ AFRAME.registerComponent("hand-pose", {
     this.from.play();
 
     const getNetworkedAvatar = el => {
-      const networkedAvatar = el.components["networked-avatar"];
+      if (!el) {
+        window.setTimeout(() => {
+          getNetworkedAvatar(this.el);
```

Comment:
What's this about? Why would `getNetworkedAvatar` call itself if it's just going to return `undefined` immediately?

Reply:
This component is on our avatars before the `networked-avatar` component is attached, so if we go all the way up the ancestors and don't find one, we restart the search at `this.el` after 1 second.

Reply:
(The next iteration of `getNetworkedAvatar` won't _immediately_ return `undefined`. Note `this` in us calling `getNetworkedAvatar(this.el)`, not `getNetworkedAvatar(el)`.)